### ISSUE TITLE
eslint angular/module-getter + angular/di false positive in __newrouter.es6

### ISSUE THREAD

**Issue by iki** [Swiip/generator-gulp-angular#908]
Generating new project with babeljs es6 and new router, and running `gulp serve`, results in eslint false positive error:
```
.../src/app/index.route.js
  11:5  error  Avoid using a variable and instead use chaining with the getter syntax  angular/module-getter
  11:5  error  You should use the function syntax for DI                               angular/di
```

This is eslint angular plugin issue Gillespie59/eslint-plugin-angular#223.

Until it's resolved in upstream, it can be fixed by disabing the respective eslint rules for https://github.com/Swiip/generator-gulp-angular/blob/master/generators/app/templates/src/app/_newrouter/__newrouter.es6#L11 by changing the line to:
```
    $router.config([ // eslint-disable-line angular/di, angular/module-getter
```

**Comment by Swiip** [comment]
Yep, mixing eslint-angular and ES6 can lead to false positive warnings. Still eslint-angular is a great plugin, I want to keep it, but a bit more of configuration could be better... (poke @Gillespie59)

**Comment by iki** [comment]
@Swiip sure, keep it, the comment above only disables the 2 rules for given line, see http://eslint.org/docs/user-guide/configuring.html#configuring-rules

**Comment by Gillespie59** [comment]
Hi, 

Thanks. The problem come from the identifier from which we execute the config method (in your case $router). 

I am working on that. We can not easily detect if $router is a service/provider or an Angular module. 

Maybe we can whitelist for every angular components
